addons/im_livechat/static/src/scss/im_livechat_form.scss

```diff
@@ -5,3 +5,14 @@
         }
     }
 }
+
+.o_livechat_layout_colors {
+    vertical-align: middle;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    .o_field_widget {
+        width: 30px !important;
```

Comment:
why `!important`? Certainly another style has more specificity, but the solution consists of making this one more specific without `!important`